coord/src/rmr/chan.c

```diff
@@ -34,13 +33,12 @@ typedef struct MRChannel {
 #include "chan.h"
 #include "rmalloc.h"
 
-MRChannel *MR_NewChannel(size_t max) {
+MRChannel *MR_NewChannel() {
```

Comment:
I guess we always use max as 0? This is why you drop it?

Reply:
yes